lib/logstash/inputs/jdbc.rb

```diff
@@ -160,6 +160,13 @@ class LogStash::Inputs::Jdbc < LogStash::Inputs::Base
   # Whether to force the lowercasing of identifier fields
   config :lowercase_column_names, :validate => :boolean, :default => true
 
+  # The character encoding used in this input. Examples include `UTF-8`
+  # and `cp1252`
+  #
+  # This setting is useful if some of your data is in `Latin-1` (aka `cp1252`)
+  # or in another character set other than `UTF-8`.
+  config :charset, :validate => ::Encoding.name_list, :default => "UTF-8"
```

Comment:
can we make here clear the purpose of this is to convert from whatever we define in :charset to utf8, so that all data is utf8?